VaticanBAV/et/BAVet116.xml

```diff
@@ -67,132 +71,150 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                            <locus from="112v" to="123r"/>
-                            <title type="complete" ref="LIT1103Anapho"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.13">
-                            <locus from="123v" to="137v"/>
-                            <title type="complete" ref="LIT1960Mashaf#AnaphoraAthanasius"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.14">
-                            <locus from="137v" to="145v"/>
-                            <title type="complete" ref="LIT2040Naakkw"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.15">
-                            <locus from="145v" to="153r"/>
-                            <title type="complete" ref="LIT1960Mashaf#AnaphoraEpiphanius"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.16">
-                            <locus from="153r" to="160v"/>
-                            <title type="complete" ref="LIT1960Mashaf#AnaphoraJohnChrysostom"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.17">
-                            <locus from="160v" to="164r"/>
-                            <title type="complete" ref="LIT1960Mashaf#AnaphoraDioscorus"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.18">
-                            <locus from="164r" to="171v"/>
-                            <title type="complete" ref="LIT1960Mashaf#AnaphoraJacobSerug"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.19">
-                            <locus from="171v" to="181r"/>
-                            <title type="complete" ref="LIT1098Anapho"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.20">
-                            <locus from="182v" to="186v"/>
-                            <title type="complete" ref="LIT2038Naakkw"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.21">
-                            <locus from="186v" to="194r"/>
-                            <title type="complete" ref="LIT1569habeka"/>
+
+                        </msItem>
+
+                        <msItem xml:id="ms_i1.2">
+                          <locus from="20v" to="24r"/>
+
+                          <msItem xml:id="ms_i1.2.1">
+                            <locus from="20v" to="22v"/>
+                            <title>Prayers of supplication for the Deceased</title>
                           </msItem>
-                          <msItem xml:id="ms_i1.22">
-                            <locus from="194r" to="197v"/>
-                            <title type="complete" ref="LIT2038Naakkw"/>
+
+                          <msItem xml:id="ms_i1.2.2">
+                            <locus from="22v" to="24r"/>
+                            <title>መስተብቍዕ፡</title>
```

Comment:
issue(s) for the new works?

Reply:
check if it may correspond to any listed here LIT4711Mastabqwe

Reply:
for 1.2.2 - BAVet108 indeed sends to LIT4711Mastabqwe, but it sounds rather like LIT6052MMIntrPrayer to me, here labeled as "መስተብቍዕ" which may therefore remain in the title field (in both 117 and 108)

Reply:
The similar prayer in 108 is actually, according to the catalogue, 11v, which has the ref LIT4270Salota.